backend/src/test/java/com/redhat/cloud/notifications/routers/NotificationResourceTest.java

```diff
@@ -1081,6 +1083,106 @@ void testNotFoundBehaviorGroupNotExists() {
         }
     }
 
+    /**
+     * Tests that when creating a behavior group, if the display name exceeds
+     * the limit set, then a corresponding error message is returned.
+     * @throws NoSuchFieldException if the field in the request class to grab
+     * the maximum value for the display name does not exist.
+     */
+    @Test
+    void testBehaviorGroupDisplayNameTooLong() throws NoSuchFieldException {
+        final Bundle bundle = this.helpers.createBundle(TEST_BUNDLE_NAME, "Bundle-display-name");
+
+        // Get the value of the "max" property for the "Size" annotation.
+        final Field classField = CreateBehaviorGroupRequest.class.getDeclaredField("displayName");
+        final Size sizeClassAnnotation = classField.getAnnotation(Size.class);
+
+        final CreateBehaviorGroupRequest createBehaviorGroupRequest = new CreateBehaviorGroupRequest();
+        createBehaviorGroupRequest.bundleId = bundle.getId();
+        createBehaviorGroupRequest.displayName = "a".repeat(sizeClassAnnotation.max() + 1);
+
+        final Header identityHeader = initRbacMock("behavior-group-display-name-too-long-account-number", "behavior-group-display-name-too-long-org-id", "user", FULL_ACCESS);
+
+        final String response = given()
+                .header(identityHeader)
+                .when()
+                .contentType(JSON)
+                .body(Json.encode(createBehaviorGroupRequest))
+                .post("/notifications/behaviorGroups")
+                .then()
+                .statusCode(400)
+                .extract()
+                .body()
+                .asString();
```

Comment:
FYI, `extract().asString()` returns the same thing.

Reply:
Aaaah, TIL! Refactored!